config.go

```diff
@@ -51,6 +51,7 @@ const (
 	defaultInactiveChanTimeout = 20 * time.Minute
```

Comment:
The travis tests are failing due to `config.go` not being correctly formatted. 

I think you'll have to run `go fmt`